myriad-scheduler/src/main/java/org/apache/myriad/configuration/NodeManagerConfiguration.java

```diff
@@ -62,8 +62,18 @@
    * Default max CPU cores for NodeManager JVM
    */
   public static final double DEFAULT_NM_MAX_CPUS = 24;
+
+  /**
+   * Default vcore ratio for NodeManager
+   */
+  public static final double DEFAULT_VCORE_RATIO = 1;
   
   /**
+   * Min vcore ratio for NodeManager
+   */
+  public static final double MIN_VCORE_RATIO = 0.1;
+
```

Comment:
We'd better have a min value in case of devide by 0 or a very small value.
0.1 is somehow a empirical value here. I think more than 10 tasks share one CPU would not work efficiently. Also I am OK if you think another min value is better.